assets/css/styles.css

```diff
@@ -603,56 +650,179 @@ Contains email and socials
   font-size: 1.25rem;
 }
 
-.footer-background {
-  background-image: url(../images/mask-group-footer.png) no-repeat;
-  bottom: 0;
-}
-
 /* HEADER AND FOOTER DISPLAY OPTIONS */
 
-header .display-desktop {
+.header .display-desktop {
   display: none;
 }
 
-header .display-mobile {
-  width: 100%;
+.header .display-mobile {
   display: flex;
-  justify-content: flex-end;
+  top: 0;
   align-items: flex-end;
 }
 
-header .display-mobile img {
+.header .display-mobile img {
   align-self: flex-end;
+  margin-right: 1rem;
 }
 
 .headline .display-desktop,
 .footer .display-desktop {
   display: none;
 }
 
+/*
+HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
+*/
+
 @media (max-width: 280px) {
-  .project-built-with li,
-  .list-competences li {
+  .header-nav .display-mobile {
+    width: 100%;
+    height: 90vh;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    visibility: hidden;
+  }
+
+  .header-nav .nav-menu {
+    display: none;
+  }
+
+  .header-nav .hamburger-menu {
+    position: fixed;
+    height: 5vh;
+    margin-top: 0%;
+  }
+
+  .header-nav .hamburger-navmenu {
+    position: fixed;
+    height: 90vh;
+    margin-top: 10%;
+  }
+
+  .project .project-built-with li,
+  .competences-card .list-competences li {
     width: 90%;
     text-align: center;
   }
+
+  footer .social .display-mobile,
+  .header .social .display-mobile {
+    width: 30%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .display-desktop {
+    display: none;
+  }
 }
 
 @media (min-width: 375px) {
   .display-desktop {
     display: none;
   }
 
-  header .display-mobile {
+  .header-nav {
+    top: 0;
+    height: 90vh;
+    position: fixed;
+  }
+
+  .menu-fixed {
+    position: fixed;
+    z-index: 2;
+  }
+
+  .navbar {
+    padding-top: 0; /* //THAT WAS 30% // */
+    overflow: hidden;
+  }
+
+  .hamburger-menu {
+    visibility: visible;
+  }
+
+  .header-nav .display-mobile {
     width: 100%;
+  }
+
+  .social .display-mobile {
+    width: 30%;
+  }
+
+  .header-nav .nav-menu {
+    display: none;
+  }
+
+  .header-nav .hamburger-navmenu {
+    height: 90vh;
+  }
```

Comment:
Kindly give your hamburger menu background a full height so it, when it's opened, it covers the full height of the page to create a good user experience.
Figma:
<img width="301" alt="Screenshot 2023-04-17 at 22 49 26" src="https://user-images.githubusercontent.com/107192076/232626921-177173f5-fa7b-43ed-a131-43070789a701.png">

Your design:
<img width="337" alt="Screenshot 2023-04-17 at 22 37 59" src="https://user-images.githubusercontent.com/107192076/232626944-ecd3efda-8479-41a9-b46f-457e39cd987c.png">

Please remember to stick to the Figma design as much as possible.

Reply:
Done, Thanks @Strangeal